tests/fsharp/perf/tests_perf.fs

```diff
@@ -0,0 +1,35 @@
+module ``FSharp-Tests-Perf``
+
+open System
+open System.IO
+open NUnit.Framework
+
+open FSharpTestSuiteTypes
+open NUnitConf
+open PlatformHelpers
+
+let testContext = FSharpTestSuite.testContext
+
+
+module Graph = 
+
+    [<Test; FSharpSuitePermutations("perf/graph")>]
+    let graph p = check (processor {
+        let { Directory = dir; Config = cfg } = testContext ()
+        
+        do! SingleTestBuild.singleTestBuild cfg dir p
```

Comment:
(If making this change is too much work - as I guess it is - then we can do it later if neccesary - one advantage of what you've done is that it makes cleanup much easier from here on)